controllers/transactionsController.js

```diff
@@ -0,0 +1,136 @@
+const Portfolio = require('../models').Portfolio;
+const League = require('../models').League;
+const Transaction = require('../models').Transaction;
+const User = require('../models').User;
+
+const get = require('../utils/request').getRequest;
+const formatDate = require('../utils/dates').formatDate;
+
+async function getStockData(key,symbol) {
+  let stockPrice;
+  try {
+    stockPrice = await get(`https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=${symbol}&apikey=${key}`);
+  } catch (error) {
+    console.log(error)
+  }
+  
+  return stockPrice.data["Global Quote"]["05. price"];
+}
+
+module.exports = {
+  getTransactions(req,res) {
+    console.log("Reached")
+    const sessionID = 'abcde12345';
```

Comment:
grab session id from the req object (req.sessionID). the code for this is in master, so you can merge it into ur branch for testing purposes!